dll/shellext/cabview/cabview.rc

```diff
@@ -0,0 +1,75 @@
+/*
+ * PROJECT:     ReactOS CabView Shell Extension
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Main resource file
+ * COPYRIGHT:   Copyright 2024 Whindmar Saksit <whindsaks@proton.me>
+ */
+
+#include <windef.h>
+#include <winuser.h>
+
+#include "resource.h"
+
+IDI_FOLDER ICON "res/folder.ico"
```

Comment:
I wonder if the Tango Icons set has a cabinet icon (the one currently used is copied from zip-folder).

Reply:
There is a cabinet (metal filing cabinet) icon in shell32 but it's used by other parts of the shell already and it does not look like a file or folder icon. Granted, the file cabinet icon Windows uses in some versions does not either.